src/applications/vaos/tests/mocks/data.js

```diff
@@ -0,0 +1,227 @@
+/**
+ * @module testing/mocks/data
+ */
+import moment from '../../lib/moment-tz';
+import omit from 'platform/utilities/data/omit';
+import { VIDEO_TYPES } from '../../utils/constants';
+
+/**
+ * Creates a mock appointment record, based on the data and version number
+ * provided.
+ *
+ * Versions currently supported:
+ * - 0: MAS appointment
+ * - 2: VAOS service appointment
+ *
+ * @export
+ * @param {Object} params
+ * @param {?string} params.id The appointment id
+ * @param {?string} params.email The email address used
+ * @param {?string} params.currentStatus The VistA status to use. Ignored in version 2
+ * @param {Number} [params.version=2] The version of the output data. Currently 0 and 2 are supported
+ * @param {?string} params.clinicFriendlyName The clinic name of the appointment (version 0 only)
+ * @param {?string} params.clinicName The regular clinic name of the appointment (version 0 only)
+ * @param {?string} params.instructionsTitle The video instructions title string
+ * @param {PPMSProvider} params.communityCareProvider The community care provider to use. Info aside from
+ *   uniqueId is discarded in version 2
+ * @param {...string} params.fields Other fields provided can be any version 2 field. Some are used to set data
+ *   on the version 0 output (kind, start, etc) and all are merged into the v2 output
+ * @returns {VAOSAppointment|MASAppointment} An appointment object in the specified format
+ */
+export function createMockAppointment({
+  id = null,
+  email = null,
+  currentStatus = null,
+  version = 2,
+  clinicFriendlyName = null,
+  clinicName = null,
+  instructionsTitle = null,
+  communityCareProvider = null,
+  timezone = null,
+  ...fields
+} = {}) {
+  const fieldsWithoutProps = omit(['email'], fields);
+  if (version === 0 && fields.kind !== 'cc') {
```

Comment:
All the logic in here is generally to to take vaos service fields and use them to set the equivalent fields in a v0 appointment. Basically a transform from a vaos service appointment to a MAS appointment.